README.md

```diff
@@ -53,18 +52,23 @@ Once you run this, should should see Major Tom respond with a `hello` message:
 
 ### Note on Major Tom Deployment Environments
 
-If you have Basic Authentication enabled on your deployment (usually true if you're accessing it over `your-org.majortom.cloud`), 
-please make sure to include the argument for providing those credentials. 
+If you have Basic Authentication enabled on your deployment
+(usually true if you're accessing it over `your-org.majortom.cloud`),
+you'll need to enter those credentials to connect.
+Enter basicauth login credentials for your Major Tom deployment (if it’s active) with `-b {username}:{password}`,
+for example: `-b kubos:password123`.
 Run `python3 run.py -h` for the full information on how to include it.
 
-If you are running the on-prem version of Major Tom, 
-you'll need to include the `--http` flag, as we currently do not support https for on prem. 
+If you are running the on-prem version of Major Tom,
+you'll need to include the `--http` flag, as we currently do not support https for on prem.
 
 ## What does this Demo Satellite do?
 
 Now that you've connected the gateway, it will automatically create a new satellite named "Space Oddity" and load in command definitions for it.
 
-You can now issue those commands to the satellite through the connected Gateway, which accomplish a wide variety of simulated tasks, all of which are explained by the description under each command.
+You can now issue those commands to the satellite through the connected Gateway,
+which accomplish a wide variety of simulated tasks.
```

Comment:
The end of this sentence reads funny to me. I think "which accomplishes" or "which can accomplish" would be more correct, if the "which accomplish" part is referring to the Gateway.

Reply:
Commands...accomplish, not Gateway...accomplishes. The subject is commands.